src/explore-education-statistics-common/src/modules/methodology/components/__tests__/MethodologyHelpAndSupportSection.test.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import * as ContactUsSection from '@common/modules/find-statistics/components/ContactUsSection';
+import MethodologyHelpAndSupportSection from '../MethodologyHelpAndSupportSection';
+import {
+  mockMethodologyPublication,
+  mockPublicationSummary,
+} from './__data__/mock-data';
+
+jest.mock('@common/modules/find-statistics/components/ContactUsSection');
+
+describe('Methodology Help and Support Section (PublicationSummary)', () => {
+  beforeEach(() => {
+    jest
+      .spyOn(ContactUsSection, 'default')
+      .mockReturnValue(<div>This is a mocked ContactUsSection</div>);
```

Comment:
I personally would like to question your assertion that all frontend tests should be testing integration. I think there's huge value in writing individual unit tests as well as integration tests. This often makes the process of writing integration tests smoother and more concise, since you don't need to unit test your component in the same file. 

This `ContactUsSection` is now used in multiple parent components. I wouldn't want to write the same suite of tests for it multiple times, as that breaks the DRY principle. Since the only "logic" per se of this parent component is that it includes the child, my intention was just to assert that and rely on the unit tests written for that child rather than repeat them. 
If you like, we could abstract the assertion into a wider "AssertThatContactUsSectionRendersCorrectly" sort of method, and just call that in all tests that include rendering it? I personally would prefer to leave as-is though.

Reply:
Have simply removed this test.